base/applications/notepad/lang/ko-KR.rc

```diff
@@ -0,0 +1,194 @@
+    EDITTEXT edt4, 119, 80, 36, 12, WS_TABSTOP | WS_GROUP | WS_BORDER
+    LTEXT "오른쪽(&R):", stc16, 159, 82, 30, 8
+    EDITTEXT edt6, 190, 80, 36, 12, WS_TABSTOP | WS_GROUP | WS_BORDER
+    LTEXT "상단(&O):", stc17, 88, 102, 30, 8
+    EDITTEXT edt5, 119, 100, 36, 12, WS_TABSTOP | WS_GROUP | WS_BORDER
+    LTEXT "하단(&B):", stc18, 159, 102, 30, 8
+    EDITTEXT edt7, 190, 100, 36, 12, WS_TABSTOP | WS_GROUP | WS_BORDER
+    LTEXT "머리말(&H):", 0x140, 8, 132, 40, 15
+    EDITTEXT 0x141, 58, 130, 173, 12, WS_BORDER | WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "꼬리말(&F):", 0x142, 8, 149, 40, 15
+    EDITTEXT 0x143, 58, 147, 173, 12, WS_BORDER | WS_TABSTOP | ES_AUTOHSCROLL
+    PUSHBUTTON "도움말", IDHELP, 8, 170, 50, 14
+    DEFPUSHBUTTON "확인", IDOK, 198, 170, 50, 14, BS_PUSHBUTTON
+    PUSHBUTTON "취소", IDCANCEL, 254, 170, 50, 14
+    PUSHBUTTON "프린터(&R)...", psh3, 310, 170, 50, 14
+END
+
+/* Dialog 'Encoding' */
+DIALOG_ENCODING DIALOGEX 0, 0, 256, 44
+STYLE DS_SHELLFONT | DS_CONTROL | WS_CHILD | WS_CLIPSIBLINGS | WS_CAPTION | WS_SYSMENU
+FONT 9, "굴림"
+CAPTION "인코딩"
+BEGIN
+    COMBOBOX ID_ENCODING, 124, 0, 125, 80, CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | WS_TABSTOP
+    LTEXT "인코딩:", 0x155, 65, 2, 41, 12
+    COMBOBOX ID_EOLN, 124, 18, 125, 80, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    LTEXT "끝 줄:", 0x156, 65, 20, 41, 12
+END
+
+/* Dialog 'Go To' */
+DIALOG_GOTO DIALOGEX 0, 0, 165, 50
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
+FONT 9, "굴림"
+CAPTION "줄 이동"
+BEGIN
+    LTEXT "줄 번호:", 0x155, 5, 12, 41, 12
+    EDITTEXT ID_LINENUMBER, 54, 10, 106, 12, ES_NUMBER
+    DEFPUSHBUTTON "확인", IDOK, 75, 30, 40, 15
+    PUSHBUTTON "취소", IDCANCEL, 120, 30, 40, 15
+END
+
+DIALOG_PRINTING DIALOG 0, 0, 160, 100
+CAPTION "지금 인쇄"
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
+FONT 9, "굴림"
+BEGIN
+    CTEXT "인쇄 작업 시작 중...", IDC_PRINTING_STATUS, 5, 10, 150, 15
+    CTEXT "(파일 이름)", IDC_PRINTING_FILENAME, 5, 35, 150, 15
+    CTEXT "페이지 %u", IDC_PRINTING_PAGE, 5, 55, 150, 15
+    PUSHBUTTON "취소", IDCANCEL, 50, 75, 60, 20
+END
+
+STRINGTABLE
+BEGIN
+    STRING_PAGESETUP_HEADERVALUE "&f"
+    STRING_PAGESETUP_FOOTERVALUE "페이지 &p"
+    STRING_NOTEPAD "Notepad"
+    STRING_ERROR "오류"
+    STRING_WARNING "경고"
+    STRING_INFO "정보"
+    STRING_UNTITLED "제목 없음"
+    STRING_ALL_FILES "모든 파일 (*.*)"
+    STRING_TEXT_FILES_TXT "텍스트 파일 (*.txt)"
+    STRING_TOOLARGE "'%s' 파일이 메모장에 사용하기에 너무 큽니다.\n\
+다른 편집기를 사용하세요."
+    STRING_NOTEXT "텍스트를 입력하지 않았습니다.\n\
+입력한 후 다시 시도하세요."
+    STRING_DOESNOTEXIST "'%s' 파일이\n존재하지 않습니다\n\n\
+새 파일을 생성하시겠습니까?"
+    STRING_NOTSAVED "'%s' 파일\n수정되었습니다\n\n\
+변경 사항을 저장하시겠습니까?"
+    STRING_NOTFOUND "'%s'을(를) 찾을 수 없습니다."
+    STRING_OUT_OF_MEMORY "메모리가 부족하여 이 작업을 완료할 수 없습니다.\n\
+하나 이상의 응용 프로그램을 닫아 사용 가능한 메모리를 늘립니다."
+    STRING_CANNOTFIND "'%s'을(를) 찾을 수 없습니다'"
+    STRING_ANSI "ANSI"
+    STRING_UNICODE "유니코드"
+    STRING_UNICODE_BE "유니코드 (빅 엔디안)"
+    STRING_UTF8 "UTF-8"
+    STRING_UTF8_BOM "UTF-8 (BOM 포함)"
```

Comment:
Other languages don't translate the names of encoding types so it would be better to follow the convention